pkg/rbac/tr.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
- Can you move to a /mocks folder to match the resolver package?
- Can you make the file name more descriptive like `pkg/rbac/mocks/tokenReview.json` ?

Reply:
Sure. I don't use that file for the tests but might be good to keep it as a sample.